src/components/atoms/Tooltip.module.css

```diff
@@ -5,8 +5,6 @@
 .content {
   composes: box from './Box.module.css';
   padding: calc(var(--spacer) / 4);
-  width: calc(100% - var(--spacer) / 3);
-  max-width: 25rem;
```

Comment:
that custom `width` calculation removal nicely solves a bunch of issues across whole app since tippy.js seems to figure everything out correctly by now. But the `max-width: 25rem;` we still need to leave in cause tippy.js tries to use full viewport width otherwise:

<img width="957" alt="Screen Shot 2021-02-22 at 11 17 47" src="https://user-images.githubusercontent.com/90316/108694983-0b99a300-7500-11eb-8d99-e9350682cb75.png">